### PR TITLE
Add introduction of a few concepts

### LINKED ISSUE #14: Duplication between Homepage and introduction
Introduction should have a different, more detailed text.

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Fixes https://github.com/OpenMS/OpenMS-docs/issues/14.

## Describe the change

Ref: https://github.com/OpenMS/OpenMS-docs/issues/4#issuecomment-1125150286.

## PR checklist

- [x] I have added description of the change I'm proposing in the OpenMS Documentation.
- [x] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [x] I have attached a screenshot of the relevant area after this change.
- [x] `CHANGELOG.md` is updated.
- [x] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+```{note}
+Note: This introduction is aimed at users new to the field of LC-MS data analysis and will introduce some basics terms
+and concepts. How to handle the data analysis, available data structures, algorithms and more are covered in the various
+subsections of this documentation.
+```
+
+# Background
+
+Proteomics and metabolomics are interdisciplinary research fields that study structure, function, and interaction of
+proteins and metabolites. They employ large-scale experimental techniques that allow acquiring data at the level of
```
Comment (greengypsy) [inline docs/introduction.md:19]: break mid-sentence

File: docs/introduction.md [hunk 2]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+cellular systems to whole organisms. One of the main analytical method to identify, characterize or quantify proteins
+and metabolites is mass spectrometry (MS) combined with chromatographic separation.
+
+In mass spectrometry-based proteomics and metabolomics, biological samples are extracted, prepared, and separated to
+reduce sample complexity. The separated analytes are ionized and measured in the mass spectrometer. Mass and abundance
+of ions are stored in mass spectra and used to identify and quantify the analytes in the sample using computational
```
Comment (greengypsy) [inline docs/introduction.md:25]: break mid-sentence

File: docs/introduction.md [hunk 3]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+- **Detector**: The last component of the mass spectrometer is the detector. It determines the abundance of ions that
+  passed through the mass analyzer. Ion intensities (a value that relates to its abundance) and the mass-to-charge ratio
+  are recorded in a mass spectrum.
+
+A sample is measured over the retention time of the chromatography typically resulting in tens of thousands of spectra. 
+The measurement of one sample is called an MS run and the set of spectra called an MS or peakmap.
+
+![](images/introduction/spectrum_peakmap.png)
+
+Left image displays spectrum with peaks (m/z and intensity values) and right image shows spectra stacked in retention
```
Comment (ghost) [inline docs/introduction.md]: The left image

Comment (ghost) [reply docs/introduction.md]: and the right image

File: docs/introduction.md [hunk 4]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+subsections of this documentation.
+```
+
+# Background
+
+Proteomics and metabolomics are interdisciplinary research fields that study structure, function, and interaction of
+proteins and metabolites. They employ large-scale experimental techniques that allow acquiring data at the level of
+cellular systems to whole organisms. One of the main analytical method to identify, characterize or quantify proteins
+and metabolites is mass spectrometry (MS) combined with chromatographic separation.
+
```
Comment (ghost) [inline docs/introduction.md:22]: Mass spectrometry combined with chromatographic separation is commonly used to identify, characterize or quantify the amount of proteins and metabolites.

File: docs/introduction.md [hunk 5]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+In mass spectrometry-based proteomics and metabolomics, biological samples are extracted, prepared, and separated to
+reduce sample complexity. The separated analytes are ionized and measured in the mass spectrometer. Mass and abundance
+of ions are stored in mass spectra and used to identify and quantify the analytes in the sample using computational
+methods. The quantity and identity of analytes can then be used, for instance, in biomarker discovery, medical diagnostics,
+or basic research.
+
+# Liquid Chromatography
+
+LC aims to reduce the complexity of the measured sample by separating analytes based on their physicochemical properties.
```
Comment (ghost) [inline docs/introduction.md:31]: Liquid Chromatography (LC)
or Liquid chromatography

File: docs/introduction.md [hunk 6]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+Separating analytes in time ensures that a manageable amount of analytes elute at the same time. In mass
+spectrometry-based proteomics, (high-pressure) liquid chromatographic separation techniques (HPLC) are methods of choice
+to achieve a high degree of separation. In HPLC, peptides are separated on a column. Solved in a pressurized liquid
+(mobile phase) they are pumped through a solid adsorbent material (stationary phase) packet into a capillary column.
+Physicochemical properties of each peptide determine how strongly it interacts with the stationary phase. The most
+commonly HPLC technique in proteomics and metabolomics uses reversed-phase chromatography (RPC) columns. RPC employs a
```
Comment (ghost) [inline docs/introduction.md]: The most common

Comment (ghost) [reply docs/introduction.md]: The most common HPLC technique in proteomics and metabolomics uses reversed-phase chromatography (RPC) columns.

File: docs/introduction.md (hunk 3 above)
Comment (jpfeuffer) [reply docs/introduction.md]: And "a spectrum"

File: docs/introduction.md [hunk 7]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
-engines like KNIME, Galaxy, WS-Pgrade, and TOPPAS via the TOPPtools concept and
-a unified parameter handling via a 'common tool description' (CTD) scheme.
-
-With pyOpenMS, OpenMS offers Python bindings to a large part of the OpenMS API
-to enable rapid algorithm development. OpenMS supports the Proteomics Standard
-Initiative (PSI) formats for MS data. The main contributors of OpenMS are
-currently the Eberhard-Karls-Universität in Tübingen, the Freie Universität
-Berlin, and the ETH Zürich.
+```{note}
+Note: This introduction is aimed at users new to the field of LC-MS data analysis and will introduce some basics terms
```
Comment (jpfeuffer) [inline docs/introduction.md]: Will this be fine on sphinx or will it be double Note Note:?

File: docs/introduction.md [hunk 8]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+  ions in a certain mass-to-charge range will reach the detector. The orbitrap is an ion trap mass analyzer (and detector)
+  that traps ions in orbital motion between a barrel-like outer electrode and a spindle-like central electrode allowing
+  for prolonged mass measurement. As a result of the prolonged mass measurements, a high mass resolution can be achieved.
+
+- **Detector**: The last component of the mass spectrometer is the detector. It determines the abundance of ions that
+  passed through the mass analyzer. Ion intensities (a value that relates to its abundance) and the mass-to-charge ratio
+  are recorded in a mass spectrum.
+
+A sample is measured over the retention time of the chromatography typically resulting in tens of thousands of spectra.
+The measurement of one sample is called an MS run and the set of spectra called an MS or peakmap.
```
Comment (jpfeuffer) [inline docs/introduction.md]:
```suggestion
The measurement of one sample is called an MS run and the set of spectra called an MS(1) map or peak map.
```

File: docs/introduction.md [hunk 9]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+
+![](images/introduction/spectrum_peakmap.png)
+
+The left image displays spectrum with peaks (m/z and intensity values) and the right image shows a spectrum stacked in
```
Comment (jpfeuffer) [inline docs/introduction.md]:
```suggestion
The left image displays spectrum with peaks (m/z and intensity values) and the right image shows spectra stacked in
```

File: docs/introduction.md [hunk 10]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+retention time yield a peak map.
```
Comment (jpfeuffer) [inline docs/introduction.md]:
```suggestion
retention time yielding a peak map.
```

File: docs/introduction.md [hunk 11]
```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
...
+
+In proteomics and metabolomics, the MS1 intensity is often used for the quantification of an analyte. Identification
+based on the MS1 mass-to-charge and the isotope pattern is highly ambiguous. To improve identification, tandem mass
+spectrometry (MS/MS) can be applied to assess the analyte substructure. To this end, the precursor ion is isolated and
+kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by collision-induced fragmentation (CID) are
+stored in an MS2 (MS/MS) spectrum and provide information that helps to resolve the ambiguities in identification.
```
Comment (jpfeuffer) [inline docs/introduction.md]:
```suggestion
stored in an MS2 (or MS/MS) spectrum and provide information that helps to resolve the ambiguities in identification.
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
Minor comments

**Pull request merged**